workflows/render-svg.yaml

```
@@ -0,0 +1,47 @@
+name: Render SVG files
+
+on:
+  push:
+    paths:
+      - man/**logo.svg
+
+jobs:
+  render-svg:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+
+      - name: Install dependencies
+        uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          packages: |
+            any::magick
+            any::rsvg
+
+      - name: Render logo SVG
+        run: |
+          library(magick)
+
+          svg_path <- list.files("man/figures", "*logo.svg", full.names = TRUE)
```

The `render-svg` workflow from ssi-dk/SCDB has recently been updated (https://github.com/ssi-dk/SCDB/commit/b7e5a4edd78a220c1435bc482f73744f8b43980f) to specifically look for `[package name]-logo.svg`.

```suggestion
          svg_path <- list.files(
            "man/figures",
            paste0(read.dcf("DESCRIPTION")[1, "Package"], "-logos.svg"),
            full.names = TRUE)
```

I would strongly recommend making these changes to this repository as well.

---

```suggestion
          svg_path <- list.files(
            "man/figures",
            paste0(read.dcf("DESCRIPTION")[1, "Package"], "-logo.svg"),
            full.names = TRUE
          )
```